tests/unit/acl-v2.tcl

```diff
@@ -0,0 +1,58 @@
+start_server {tags {"acl external:skip"}} {
+    set r2 [redis_client]
+    test {Test basic multiple selectors} {
+        r ACL SETUSER selector-1 on -@all resetkeys nopass
+        $r2 auth selector-1 password
+        catch {$r2 ping} _
+        catch {$r2 set write::foo bar} _
+        catch {$r2 get read::foo} _
```

Comment:
what's the purpose of these? maybe you wanna use `assert_error` (to verify you're getting the right error)